pkgs/applications/graphics/viewnior/default.nix

```
@@ -1,18 +1,24 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gtk2, libpng, exiv2, lcms
+{ stdenv, fetchFromGitHub, meson, ninja, pkgconfig, desktop-file-utils, gtk2, libpng, exiv2, lcms
 , intltool, gettext, shared-mime-info, glib, gdk-pixbuf, perl}:
 
 stdenv.mkDerivation rec {
   pname = "viewnior";
-  version = "1.6";
+  version = "1.7";
 
   src = fetchFromGitHub {
-    owner = "xsisqox";
+    owner = "hellosiyan";
     repo = "Viewnior";
     rev = "${pname}-${version}";
-    sha256 = "06ppv3r85l3id4ij6h4y5fgm3nib2587fdrdv9fccyi75zk7fs0p";
+    sha256 = "0y4hk3vq8psba5k615w18qj0kbdfp5w0lm98nv5apy6hmcpwfyig";
   };
 
-  nativeBuildInputs = [ autoreconfHook ];
+    nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    desktop-file-utils
+  ];
+
   buildInputs =
     [ pkgconfig gtk2 libpng exiv2 lcms intltool gettext
       shared-mime-info glib gdk-pixbuf perl
```

i believe pkgsconfig, intltool, and gettext usually go in naviteBuildInputs
```suggestion
  buildInputs =
    [ gtk2 libpng exiv2 lcms
      shared-mime-info glib gdk-pixbuf perl
```